src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1417,8 +1421,8 @@ static bool addTriggeredSubscriptions_noCache
           lastNotification,
           sub.hasField(CSUB_FORMAT) ? stringToFormat(STR_FIELD(sub, CSUB_FORMAT)) : XML,
           STR_FIELD(sub, CSUB_REFERENCE),
-          //subToAttributeList(sub)); siganture changed in TriggeredSubscription() constructor in 0.24.0
-          subToAttributeList(sub), NULL);
+          //subToAttributeList(sub)); signature changed in TriggeredSubscription() constructor in 0.24.0
+          subToAttributeList(sub), "");
```

Comment:
This here was a failure, a remain from the TriggeredSubscription accepted a pointer to the subscription in the cache.
Made broker crash if started with -noCache

NTC